dags/utils/dataproc.py

```diff
@@ -279,13 +288,19 @@ def moz_dataproc_pyspark_runner(parent_dag_name=None,
     :type master_disk_type: str
     :param master_disk_size:              Disk size for the master node
     :type master_disk_size: int
+    :param master_num_local_ssds : Number of local SSDs to mount
```

Comment:
Thanks for adding this to all 3 runner methods. Can we add more description as to why we'd ever want to set these, and what a default of 0 means

Reply:
Added better doc strings: e04d7375